### DISCUSSION TITLE
Need little help with mining in arduino using esp 8266 as a slave.

### DISCUSSION THREAD

**Discussion by MuhammedRaslan** [revoxhere/duino-coin#823] [Q&A]
I have seen the diagram but I'm not fully understood.
I need know that whether you have to upload code in every arduinos or just upload only in esp 8266.
And, is it necessary to use the 3.3v converter or can I just plug power through usb.
It will be a big help to me if you reply. 
:)

**Comment by Raabitco** [comment]
Hi there, the code made for arduino uno or ESP32 or ESP8266, you choose the code for the devise you have, each devise have its own code.
Regarding your 2nd question, what I use for mining is ESP8266, I use normal usb 5V because ESP2866 reduce the power to 3.3v, also, 1st try 1 arduino and see how its go. 
Hope this will be helpful
